dist/tools/cpy2remed/cpy2remed.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+#cpy2remed - copy to removable media
+#$1 contains generated hexfile
+#$2 contains directory name for this particular nucleo board which could be
+#   set in the board makefile.include using variable DIR_NAME_AT_REMED
+
+HEXFILE="$1"
+DEV_DIR="$2"
+
+REMED_MOUNT_PATH="${REMED_MOUNT_PATH:-/media/$USER/}$DEV_DIR/"
```

Comment:
What does `${REMED_MOUNT_PATH:-` do?

Reply:
It checks if the variable is unset or null and substitute text (see [Shell parameter expansion](http://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html). This change was suggested in my previous code of elf2uf2 (PR #17348) by one of reviewers.

Reply:
I looked at this line once again an I should move ```}``` to the end of line. I tested this change and script works.

Reply:
Why not just

    REMED_MOUNT_PATH="/media/$USER/$DEV_DIR/"

Reply:
This line should works to. But, as I write in last comment one of the reviewers from PR #17348 suggest usage of this construction. I'm not an expert in bash ... so I use this same construction.